.github/workflows/issue-comment.yml

```diff
@@ -0,0 +1,40 @@
+# issue-comment.yml
+# mark the status of the issue as waiting for repliers (contributor should reply) or waiting for author (issue author should reply)
+name: Add issues workflow labels
+
+on:
+  issue_comment:
+    types: [ created ]
+
+jobs:
+  add-label-if-is-author:
+    runs-on: ubuntu-latest
+    if: (github.event.issue.user.id == github.event.comment.user.id) && !github.event.issue.pull_request && (github.event.issue.state == 'open')
+    steps:
+      - name: Add require handle label
+        uses: actions-cool/issues-helper@v2
+        with:
+          actions: 'add-labels'
+          labels: 'waiting for repliers'
+
+      - name: Remove require reply label
+        uses: actions-cool/issues-helper@v2
+        with:
+          actions: 'remove-labels'
+          labels: 'waiting for author'
```

Comment:
Hi, @zhicheng-ning, can we do the remove-label step first than do the add-label step?

Does this influence how the event show in timeline?

<img width="853" alt="image" src="https://user-images.githubusercontent.com/32434520/195263690-3f23699c-7012-4867-b8b1-875e90b9946f.png">

Reply:
It doesn't influence the order the event show in timeline.

Reply:
Got it, thank you for testing.